assets/search/components/ContentSearchResults.jsx

```diff
@@ -125,7 +125,7 @@ class ContentSearchResultsComponent extends React.Component {
                             {!this.props.showTotalLabel ? null : (
                                 <React.Fragment>
                                     <span className="search-results-info__text flex-column">
-                                        <span>{gettext('search results for: ')}{this.props.searchParams.query}</span>
+                                        <span>{gettext('search results for: ') + this.props.searchParams.query}</span>
```

Comment:
this won't have the space when it's removed during translation, would need something like:
```
gettext('search results for:') + ' ' + this.props.searchParams.query
```
to be sure the space is there